paper/paper.md

```diff
@@ -0,0 +1,38 @@
+
+---
+title: "covidregionaldata: An R Package for subnational and national level COVID-19 data"
+authors:
+  - name: Joseph PG. Palmer
+    affiliation: 1
+affiliations:
+ - name: Royal Holloway University of London
+   index: 1
+date: "27 April 2021"
+bibliography: paper.bib
+tags:
+  - R
+  - COVID-19
+
+output: articles::joss_article
+csl: apa.csl
+journal: JOSS
+link-citations: yes
+---
+
+# Summary 
+
+`covidregionaldata` is an R [@Rdev:2013] package to interface subnational and national level COVID-19 data, providing cleaned and checked COVID-19 case and, where available, deaths, recoveries and hospitalisation data for all countries supported. All package code is archived on Zenodo [@covidregionaldata2020] and [Github](https://github.com/epiforecasts/covidregionaldata) Currently `covidregionaldata` provides subnational data collated by official government bodies for 14 countries including the UK, India, USA, and Brazil. In addition, national level data is provided from the World Health Organisation (WHO) and European Centre for Disease Prevention and Control (ECDC). In addition, we also provide an interface to data curated by John Hopkins University (JHU), used for their [2019 Novel Coronavirus Visual Dashboard](https://github.com/CSSEGISandData/COVID-19), and Google's [covid-19-open-data](https://github.com/GoogleCloudPlatform/covid-19-open-data). The package reduces the burden of having to process data from different sources, allowing researchers to focus their efforts on research. 
+
+# Statement of need 
+
+The onset of the COVID-19 pandemic in late 2019 has placed immense pressures on the public health community to not only advise governments around the world on how this disease might impact communities, but to also suggest what policies would most reduce transmission. Mathematical models forecasting case numbers, deaths and hospitalisations have proven to be a key tool for guiding government policy in many countries, yet generating valuable insights requires that the incidence data used is reliable and consistent.  
```

Comment:
This is a question by a non-epidemiologist reader: does "incidence" only refer to the number of new (confirmed) cases on a particular day, or can it more generally refer to the number of (attributed) deaths on a particular day? I think some of the downstream users use death data for their models but it's not clear (non-epidemiologist) whether that is included in "incidence".